lib/experimental/Widgets/Content/IconText/index.tsx

```diff
@@ -21,21 +21,22 @@ export const IconText = forwardRef<HTMLDivElement, IconTextProps>(
     return (
       <div
         ref={ref}
-        className="flex flex-row items-center gap-1 text-f1-foreground-secondary"
+        className="flex flex-row items-start gap-1 text-f1-foreground-secondary"
       >
-        <Icon icon={iconSvg} size={"sm"} />
-        {texts.map((text, index) => {
-          return (
-            <div className="flex flex-row items-center justify-center gap-1">
-              {index > 0 && (
-                <div className="h-[0.15rem] w-[0.15rem] rounded-full bg-f1-foreground" />
-              )}
-              <p key={index} className="font-medium text-f1-foreground">
-                {text}
-              </p>
-            </div>
-          )
-        })}
+        <div className="relative top-0.5">
+          <Icon icon={iconSvg} size="sm" />
+        </div>
+        <p className="font-medium text-f1-foreground">
+          {texts.map((text, index) => (
+            <span
```

Comment:
We should probably add a `key` prop here

Reply:
thanks!